### PR TITLE
en,zh: Add warning about `spec.tikv.evictLeaderTimeout` field

### PR DESCRIPTION
**Pull request by KanShiori**
### Which TiDB Operator version(s) do your changes apply to? (Required)

Add warning about `spec.tikv.evictLeaderTimeout` field.

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
https://github.com/tikv/tikv/pull/10364

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -553,6 +553,10 @@ spec:
     evictLeaderTimeout: 10000m
 ```
 
+> **警告：**
+>
+> 如果使用 TiKV 版本小于 4.0.14，或者小于 5.0.3，由于 [TiKV 的 bug](https://github.com/tikv/tikv/pull/10364)，需要将 `spec.tikv.evictLeaderTimeout` 的值设置的尽可能大。
```
Comment (qiancai) [inline zh/configure-a-tidb-cluster.md:558]: 请问 “尽可能大” 是至少要设置多大呢？

Comment (KanShiori) [reply zh/configure-a-tidb-cluster.md:558]: @DanielZhangQD  这个值怎样描述好些呢

Comment (KanShiori) [reply zh/configure-a-tidb-cluster.md]: https://github.com/pingcap/docs-tidb-operator/pull/1281/commits/6c62cc1b3a04beebc60d92f2c195f80b3197a727

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -571,6 +571,12 @@ spec:
     evictLeaderTimeout: 10000m
 ```
 
+> **Warning:**
+>
+> If TiKV's version is earlier than 4.0.14, or is earlier than 5.0.3, due to [a bug of TiKV](https://github.com/tikv/tikv/pull/10364) please set `spec.tikv.evictLeaderTimeout` as large as possible to ensure that all region leaders on the TiKV Pod can be evicted within time.
+>
+> It is recommended to set the value greater than `1500m`.
```
Comment (qiancai) [inline en/configure-a-tidb-cluster.md:578]:
```suggestion
> If the TiKV version is earlier than 4.0.14 or 5.0.3, due to [a bug of TiKV](https://github.com/tikv/tikv/pull/10364), you need to configure the timeout `spec.tikv.evictLeaderTimeout` as large as possible to ensure that all Region leaders on the TiKV Pod can be evicted within the timeout. If you are not sure about the proper value, greater than '1500m' is recommended.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**